### PR TITLE
add statement

### PR DESCRIPTION
**Pull request by Angelica137**
Statement class was created to display the required min payment for that month

### CODE REVIEW

File: scripts/statement.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+class Statement:
+    def __init__(self, balance, minMonthPymtRate):
+        self._balance = balance
+        self._minMonthPymtRate = minMonthPymtRate
+        self._minPmtThisMonth = 0
+
+    def getMinPmt(self, balance, minMonthPymtRate):
+        minPmtThisnMonth = MinMonthPymt(balance, minMonthPymtRate)
+        return minPmtThisnMonth.calcMinMonPymt(minMonthPymtRate, balance)
```
Comment (Edward-Phillips) [inline scripts/statement.py:12]: Nice, ignorant python question, do you have to pass self to access the things defined in init?

File: tests/test_statement.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+def test_generate_monthly_statement():
+    statementMonth0 = Statement(42, 0.04)
+    assert statementMonth0.getMinPmt(42, 0.04) == 1.68
+
+
+def test_print_statement():
+    statementMonth0 = Statement(42, 0.04)
+    assert statementMonth0.printStatement(
+        42, 1.68) == "Balance: 42; Minimum Payment: 1.68"
```
Comment (Edward-Phillips) [inline tests/test_statement.py:12]: nice

**Review by Edward-Phillips — APPROVED** [review]
LGTM

File: scripts/statement.py (hunk 1 above)
Comment (Angelica137) [reply scripts/statement.py:12]: yes, you have, I was surprised about this too.

**Pull request merged**